src/classes/BrainStateClass.ts

```diff
@@ -1,10 +1,10 @@
-import UseCaseEnum from './UseCaseEnum';
+import UseCase from '../interfaces/useCase';
 
 class BrainState {
   chatId:number;
 
   interruptedUseCase:Boolean;
 
-  activeUseCase:UseCaseEnum;
+  activeUseCase: UseCase;
```

Comment:
Why do you Import an Instance of an Interface? This should have been the deciding factor for routing

Reply:
Das ist outdated, es gibt keinen Brain State mehr. Der aktive Use Case ist in `Olaf` gespeichert, weil er nur da verwendet wird, und das UseCaseEnum wird nicht mehr benötigt.